zulipterminal/model.py

```diff
@@ -57,6 +57,13 @@ class ServerConnectionFailure(Exception):
     pass
 
 
+def sort_streams(streams: List[List[str]]) -> None:
+    """
+    Used for sorting model.pinned_streams and model.unpinned_streams.
+    """
+    streams.sort(key=lambda s: s[0].lower())
+
+
```

Comment:
I hope it is okay to put it here because I can't make it a class method. I don't see a reason to make this a static method, so this was the only choice.
Also, I don't think I understand why `_stream_info_from_subscriptions()` is a static method? Why would you need static methods at all when there is only one `Model` instance?

Reply:
This is fine here for now. I'm expecting there may be more refactoring of stream data in the near future, so just having it local to the model is sufficient.

A static method is just in a different namespace. I think the only difference we could make would be to have it with a leading underscore, but we're not really consistent otherwise right now.

I think `_stream_info_from_subscriptions` might be inlined in future. The advantage is that the individual functions can be tested, and static methods are guaranteed not to modify any data in an object, except potentially that passed in - which can be caught with a type checker, ideally.